.gitignore

```diff
@@ -72,6 +72,7 @@ tools/FAKE.Core
 tools/SourceLink.Fake
 tools/xunit.runners
 *.ncrunch*
+*.GhostDoc.xml
```

Comment:
Someone would need to convince me of the value of GhostDoc -  I have nothing but bad memories from the time I last touched it.

:thumbsup: to ignoring for now